### PR TITLE
5329 control play music

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
Control Play Music Fragment
link redmine : https://edu-redmine.framgia.vn/issues/5329

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/vn/kienphung/music_52/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -112,7 +112,9 @@ public void onItemClick(int position) {
 
     @Override
     public void onItemClick(List list, int position) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/vn/kienphung/music_52/ui/home/HomeFragment.java]: List này là List gì?

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/vn/kienphung/music_52/ui/playmusic/PlayMusicFragment.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.vn.kienphung.music_52.R;
+
+public class PlayMusicFragment extends Fragment {
+    private static final String TAG = "PlayMusicFragment";
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/vn/kienphung/music_52/ui/playmusic/PlayMusicFragment.java:14]: remove it

**Comment by phungvankien1997** [comment]
ready

**Pull request merged**